### PR TITLE
Add tests for addskill and its parser

### PR DESCRIPTION
**Pull request by RyanCheungJF**
Added the overdue tests for command addskill which was added in v1.3.

### CODE REVIEW

File: src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java [hunk 1]
```diff
@@ -1,4 +1,107 @@
...
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code AddSkillCommand}
+ */
 public class AddSkillCommandTest {
+
+    private Model model = new ModelManager(getTypicalLinkedout(), new UserPrefs());
+
+    @Test
+    public void execute_addExistingSkills_success() {
+        Applicant firstApplicant = model.getDefaultApplicantList().get(0);
```
Comment (zzhikai) [inline src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java:36]: Can consider using same way of getting applicants as `EditCommandTest` to reduce confusion.
Screenshot 2022-04-05 at 5 47 19 PM

File: src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java [hunk 2]
```diff
@@ -1,4 +1,107 @@
...
+
+        Model expectedModel = new ModelManager(new Linkedout(model.getLinkedout()), new UserPrefs());
+        expectedModel.setApplicant(model.getDefaultApplicantList().get(0), editedApplicant);
+
+        assertCommandSuccess(addSkillCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_addNewSkills_success() {
+        Applicant firstApplicant = model.getDefaultApplicantList().get(0);
```
Comment (zzhikai) [inline src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java:53]: Same as above

**Review by zzhikai — APPROVED** [review]
LGTM, just a minor comment.

File: src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java (hunk 1 above)
Comment (RyanCheungJF) [reply src/test/java/seedu/linkedout/logic/commands/AddSkillCommandTest.java:36]: Thanks for the suggestion. I have fixed both instances.

**Pull request merged**